samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,178 @@
+* One additional file for handling Mesh models, :file:`model_handler.c`.
+
+Provisioning
+============
+
+The provisioning is handled by the :ref:`bt_mesh_dk_prov`, and can be done with the `nRF Mesh mobile app`_.
+It is also possible to configure the models with the app.
+
+Models
+======
+
+The following table shows the Mesh sensor server composition data for this sample:
+
+   +---------------+
+   |  Element 1    |
+   +===============+
+   | Config Server |
+   +---------------+
+   | Health Server |
+   +---------------+
+   | Sensor Server |
+   +---------------+
+
+The models are used for the following purposes:
+
+* Config Server allows configurator devices to configure the node remotely.
+* Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
+  These callbacks trigger blinking of the LEDs.
+* Mesh sensor server provides sensor data to one or more :ref:`Mesh sensor client(s) <bt_mesh_sensor_cli_readme>`.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the ``TEMP_NRF5`` temperature sensor, and :ref:`dk_buttons_and_leds_readme` to detect button presses.
+
+Requirements
+************
+
+* One of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52DK|
+
+* Smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
+
+  * `nRF Mesh mobile app for Android`_
+  * `nRF Mesh mobile app for iOS`_
+
+* :ref:`bluetooth_mesh_sensor_client` sample application programmed on a separate device and configured according to the Mesh :ref:`bt_mesh_sensor_cli_readme` sample's :ref:`bluetooth_mesh_sensor_client_testing` guide.
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+
+   In addition, button 1 is used to simulate presence detected.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/sensor_server`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_sensor_server_testing:
+
+Testing
+=======
+
+.. important::
+   The :ref:`bt_mesh_sensor_srv_readme` sample cannot demonstrate any functionality on its own, and needs a device with the :ref:`bluetooth_mesh_sensor_client` sample running in the same Mesh network.
+   Before testing Mesh :ref:`bt_mesh_sensor_srv_readme`, go through the Mesh :ref:`bt_mesh_sensor_cli_readme`'s :ref:`bluetooth_mesh_sensor_client_testing` guide with a different board.
+
+After programming the sample to your board, you can test the device by provisioning and configuring it for communication trough the Mesh models.
+This can be done using a smartphone with Nordic Semiconductor's nRF Mesh app installed.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the Mesh network.
```

Comment:
```suggestion
The provisioning assigns an address range to the device, and adds it to the mesh network.
```
I know it can be confusing regarding Mesh vs mesh. Sometimes we might be in doubt ourselves LOL
Mesh in general terms like here (mesh network) is not capitalized. Capital letter only used when part of a name/term, like (Bluetooth) Mesh sensor server or nRF Mesh app.